tgui/packages/tgui/interfaces/CentcomPodLauncher.js

```diff
@@ -793,6 +793,7 @@ class PresetsPage extends Component {
     for (let i = 0; i < presets.length; i++) {
       if (presets[i].id === deleteID) {
         presets.splice(i, 1);
+        i--;
```

Comment:
Can't you just `presets = presets.filter(preset => preset.id !== deleteID)` here ?

Reply:
I've committed it, @tralezab please re-test.